src/Entity/User.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * @ORM\Entity()
+ */
+class User implements UserInterface
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue(strategy="UUID")
+     * @ORM\Column(type="string")
+     *
+     * @var string $id
+     */
+    protected $id;
+
+    /**
+     * @ORM\Column(type="string")
+     *
+     * @var string
+     */
+    protected $username;
+
+    /**
+     * @ORM\Column(type="string")
+     *
+     * @var string
+     */
+    protected $password;
+
+    /**
+     * @var string|null
+     */
+    protected $salt;
+
+    /**
+     * @ORM\Column(type="array")
+     *
+     * @var array<int, string>
+     */
+    protected $roles;
+
+    /**
+     * @param array<int, string> $roles
+     */
+    public function __construct(string $username, string $password, ?string $salt, array $roles)
+    {
+        $this->username = $username;
+        $this->password = $password;
+        $this->salt = $salt;
+        $this->roles = $roles;
+    }
+
+    /**
+     * @inheritDoc
```

Comment:
Any reason for adding this inheritDoc?

Reply:
There is a doc comment in the UserInterface that this references to.